src/markdown-pages/add-ons/ekco.md

```diff
@@ -112,7 +112,7 @@ The duration can be adjusted by changing the `ekco.envoyPodsNotReadyDuration` pr
 
 ### TLS Certificate Rotation
 
-EKCO supports automatic certificate rotation for the [registry add-on](/docs/install-with-kurl/setup-tls-certs#registry) and the [Kubernetes control plane](/docs/install-with-kurl/setup-tls-certs#kubernetes-control-plane) since version 0.5.0 and for the [KOTS add-on](/docs/install-with-kurl/setup-tls-certs#kots-tls-certificate-renewal) since version 0.7.0.
+EKCO supports automatic certificate rotation for the [registry add-on](/docs/install-with-kurl/setup-tls-certs#registry) and the [Kubernetes control plane](/docs/install-with-kurl/setup-tls-certs#kubernetes-control-plane) since version 0.5.0 and for the KOTS add-on since version 0.7.0. For more information about the KOTS add-on, see [KOTS add-on](/docs/add-ons/kotsadm). For more information about automatic certificate rotation in Replicated app manager (which uses the KOTS add-on), see [Using TLS Certificates](https://docs.replicated.com/vendor/packaging-using-tls-certs) in the Replicated documentation.
```

Comment:
Suggestions
- There's lots of links in the paragraph, which is not necessarily a bad thing, but I'm not immediately certain which one I would want to click on first. It looks like it's probably that last one about how to actually set up the automatic certificate rotation. Are the other links needed?

  If so, it could be useful to add one or two more sentences to explain what I would need to do if I go to those links. For example, it looks like the control plane link sends me to a section about checking the expiration date on a cert. How does that task fit into the larger task of setting up automatic rotation?
- "since version X" might be easier to translate as "for the KOTS add-on version X and later"
- Replicated app manager --> the Replicated app manager

Reply:
The control plane and registry add-on info is for the wider kURL audience, and explaining it further is outside the scope of this SC story. But I was able to simplify the linking, and I think this should work for the purposes of migrating the app manager related content. Good suggestions, thanks!